integrated_channels/integrated_channel/models.py

```diff
@@ -73,6 +96,18 @@ def channel_worker_user(self):
         worker_username = self.channel_worker_username if self.channel_worker_username else 'enterprise_channel_worker'
         return User.objects.filter(username=worker_username).first()
 
+    @property
+    def customer_catalogs_to_transmit(self):
+        """
+        Return the list of EnterpriseCustomerCatalog objects.
+        """
+        catalogs_list = []
+        if self.catalogs_to_transmit:
+            catalogs_list = EnterpriseCustomerCatalog.objects.filter(
+                uuid__in=list(set([uuid.strip() for uuid in self.catalogs_to_transmit.split(",") if uuid.strip()]))
```

Comment:
We should avoid this loop and set and then list conversion at multiple places instead make a method and put these operations inside it.